packages/framework/presence/src/test/mockEphemeralRuntime.ts

```diff
@@ -51,8 +51,10 @@ export class MockEphemeralRuntime implements IEphemeralRuntime {
 
 	public readonly listeners: {
 		connected: ((clientId: ClientConnectionId) => void)[];
+		disconnected: (() => void)[];
 	} = {
 		connected: [],
+		disconnected: [],
```

Comment:
Should not be needed with the change suggested to relocate `removeClientConnectionId` call.

Reply:
nevermind